crates/biome_configuration/src/analyzer/linter/rules.rs

```diff
@@ -3436,6 +3436,10 @@ pub struct Nursery {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub use_sorted_classes:
         Option<RuleFixConfiguration<biome_js_analyze::options::UseSortedClasses>>,
+    #[doc = "Enforce ordering of CSS properties."]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I know sorting alphabetically may have issues ([example](https://github.com/prettier/prettier/issues/2502#issuecomment-315713654)) but I'd really appreciate having an option for it. For me it's much more useful than the "recess-order" configuration. Would it be difficult to add support for it?

Reply:
What's a "recess-order"?

Reply:
The grouped order that `stylelint-config-recess-order` uses by default, and the one this PR was based on.

Reply:
I'd appreciate having an option, too.

It [looks like](https://github.com/biomejs/biome/pull/4063#issue-2544682488) `stylelint-config-recess-order` was chosen because it's the [most popular](https://level222.github.io/stylelint-order-comparison/public/) shared config. However, the Stylelint order plugin [has a dedicated rule](https://github.com/hudochenkov/stylelint-order/tree/master?tab=readme-ov-file#rules) for alphabetically sorting, and that may not have been taken into account. It's difficult to compare its popularity with those of the shared configs, as they have their own npm download stats. 

The dedicated rule is used in [2.2k public configs](https://github.com/search?q=%22order/properties-alphabetical-order%22:+true&type=code), so a lot of people prefer to sort alphabetically. Probably because there are 600+ properties, with more added all the time, so it's hard to maintain meaningful categories for all these.